src/index.js

```diff
@@ -65,22 +65,47 @@ const poll = () => {
     })
 }
 
+const clearLog = () => {
+  const request = new Request('/clearLog', {
+    method: 'POST',
+  })
+
+  fetch(request)
+    .then(response => {
+      if (response.status === 200) {
+        return response.json()
+      } else {
+        throw new Error('Something went wrong on api server!')
+      }
+    })
+    .then(({ log }) => {
+      display(log)
+    }).catch(error => {
+      console.log(error)
+    })
+}
+
 poll()
 setInterval(poll, 1000)
 
 </script>
+<div>  <label for="name">Enter your name:</label </div>
 <div>
-   <input id="name" type=text placeholder="enter your name.."/>
-   <input type="button" value="Click me" onClick='roll()' id='roller' />
+   <input id="name" type=text />
+   <input type="button" value="Roll D20" onClick='roll("d20")' id='roller' />
+   <input type="button" value="Roll D8" onClick='roll("d8")' id='roller' />
+   <input type="button" value="Roll D6" onClick='roll("d6")' id='roller' />
+   <input type="button" value="Clear Log" onClick='clearLog();poll()' id='roller' />
    <div id="log" />
 </div>`)
 })
 
-const rollLog = []
+var rollLog = []
 
 app.post('/roll', (req, res) => {
   const { body: { die, name } } = req
   const rolled = roll(die)
+  const time = getTime()
```

Comment:
```suggestion

```

Reply:
unused - suggest that feature goes in its own PR

Reply:
yes true

Reply:
its still here?